app/src/main/java/com/example/inventory/AddItemFragment.kt

```diff
@@ -114,13 +126,23 @@ class AddItemFragment : Fragment() {
                 this.binding.itemName.text.toString(),
                 this.binding.itemPrice.text.toString(),
                 this.binding.itemCount.text.toString(),
-                this.binding.itemSum.text.toString()
+                this.binding.itemSum.text.toString(),
+                this.imagePath.toString(),
             )
             val action = AddItemFragmentDirections.actionAddItemFragmentToItemListFragment()
             findNavController().navigate(action)
         }
     }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        super.onActivityResult(requestCode, resultCode, data)
+        if (resultCode == RESULT_OK && requestCode == pickImage) {
```

Comment:
Could you leave a comment/link for what this function is doing and what these codes refer to? Just for the team's learning reference. Otherwise PR lgtm to merge and include for demo.